src/common/404/PlayNotFound.jsx

```diff
@@ -0,0 +1,18 @@
+import { Link } from 'react-router-dom';
+import { ReactComponent as Image404 } from "images/img-404.svg";
+import "./404.css";
+
+const PageNotFound = () => {
+
+  return(
```

Comment:
It looks the same as the PageNotFound component. The content should be different.

Reply:
yes sir its for just testing purposes, once the functionality is working perfectly then i will change the component